src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -0,0 +1,85 @@
+package com.bootcamp.demo.model;
+
+import javax.validation.constraints.NotNull;
```

Comment:
Some notes regarding this:
https://stackoverflow.com/questions/34094039/using-notnull-annotation-in-method-argument (the first reply)
https://www.baeldung.com/javax-validation
https://www.baeldung.com/java-bean-validation-not-null-empty-blank